tests/evaluate/test_default_evaluator.py

```diff
@@ -4168,26 +4166,29 @@ def test_xgboost_model_evaluate_work_with_shap_explainer():
     X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.33, random_state=42)
 
     xgb_model = xgboost.XGBClassifier()
-    with mlflow.start_run():
+    with mlflow.start_run() as run:
         xgb_model.fit(X_train, y_train)
 
-        eval_data = X_test
-        eval_data["label"] = y_test
+    logged_models = mlflow.search_logged_models(
+        filter_string=f"source_run_id='{run.info.run_id}'", output_format="list"
+    )
+    model_uri = logged_models[0].model_uri
```

Comment:
I'm not sure if this works. The SQL backend doesn't support filter_string yet.

Reply:
Seems it works